### DISCUSSION TITLE
Lenovo Legion toolkit -Unsupported laptop

### DISCUSSION THREAD

**Discussion by Lapingaman32** [BartoszCichecki/LenovoLegionToolkit#1032] [Q&A] [answered]
I want to use the Lenovo Toolkit, but it does not support my laptop "This device is not compatible with the Lenovo Legion Toolkit". I have a ThinkBook 16p G2 ACH - Type 20YM. The specs are: AMD Ryzen 9 5900HX with Radeon Graphics + NVIDIA RTX 3060..

Am using Windows 11, but also use Windows 10 which I prefer.

Is there a compatible (older version) of the Lenovo Toolkit. If not, how can I contact the program creator.
Thank you.

**Comment by BartoszCichecki** [comment]
https://github.com/BartoszCichecki/LenovoLegionToolkit#arguments

**Reply by Lapingaman32** [reply]
Thanks, but I have no idea how to understand what the link you provided means, how to understanding it, or how to proceed. It's too technical for me.

**Reply by BartoszCichecki** [reply, marked as answer]
Sorry, can't help more. I gave you link to a section of the readme that describes in detail what you need to do if you want to run LLT on an unsupported device (which isn't recommended). If that's still unclear, sorry, but I don't provide support for unsupported devices.

**Reply by Lapingaman32** [reply]
No problem. I'll see if I an suss it out. Thanks again.
